brainscore_language/models/lm1b/__init__.py

```diff
@@ -0,0 +1,308 @@
+        return next_word
+    
+    def output_to_representations(self, layer_representations, stimuli_coords):
+        """Format layer representations as a neuroid assembly"""
+        
+        # Re-format layer_representations to contain region, recording type, and layer
+        layer_reps_mapping = {}
+        for layer_i, representations in enumerate(layer_representations):
+            recording_target, recording_type = self.neural_recordings[layer_i]
+            layer = self.region_layer_mapping[recording_target]
+            
+            key = (recording_target, recording_type, layer)
+            layer_reps_mapping[key] = representations
+        layer_representations = layer_reps_mapping
+
+        representation_values = np.concatenate(list(layer_representations.values()), axis=-1)  # concatenate along neuron axis
+        neuroid_coords = {
+            'layer': ('neuroid', np.concatenate([[layer] * values.shape[-1]
+                                                 for (recording_target, recording_type, layer), values
+                                                 in layer_representations.items()])),
+            'region': ('neuroid', np.concatenate([[recording_target] * values.shape[-1]
+                                                  for (recording_target, recording_type, layer), values
+                                                  in layer_representations.items()])),
+            'recording_type': ('neuroid', np.concatenate([[recording_type] * values.shape[-1]
+                                                          for (recording_target, recording_type, layer), values
+                                                          in layer_representations.items()])),
+            'neuron_number_in_layer': ('neuroid', np.concatenate(
+                [np.arange(values.shape[-1]) for values in layer_representations.values()])),
+        }
+        neuroid_coords['neuroid_id'] = 'neuroid', functools.reduce(defchararray.add, [
+            neuroid_coords['layer'][1], '--', neuroid_coords['neuron_number_in_layer'][1].astype(str)])
+        
+        representations = NeuroidAssembly(
+            representation_values, # 1024
+            coords={**stimuli_coords, **neuroid_coords},
+            dims=['presentation', 'neuroid'])
+        return representations
+        
+    def _load_encoder(self):   
+        """Download model resources, if not available, and build model from checkpoints"""
+
+        # Downloading resources (checkpoint) if not available
+        cur_dirname = os.path.dirname(os.path.realpath(__file__))
+        resources_dir = os.path.join(cur_dirname, "resources")
+        if not os.path.isdir(resources_dir):
+            os.mkdir(resources_dir)
+            downloading_iter = tqdm(RESOURCES, desc='Downloading resources for LM1B.')
+            for res in downloading_iter:
+                file_name = res.split("/")[-1]
+                file_path = os.path.join(resources_dir, file_name)
```

Comment:
suggest using pathlib
```suggestion
        cur_dirname = Path(__file__).parent
        resources_dir = cur_dirname / "resources"
        if not resources_dir.is_dir():
            resources_dir.mkdir()
            downloading_iter = tqdm(RESOURCES, desc='Downloading resources for LM1B.')
            for resource in downloading_iter:
                file_name = resource.name
                file_path = resources_dir / file_name
```

Also I would move all the downloading into its own method.